src/test/run-pass/macro-at-most-once-rep.rs

```diff
@@ -12,13 +12,19 @@ macro_rules! foo {
     ($(a)?) => {}
 }
 
+macro_rules! baz {
+    ($(a),?) => {} // comma separator is meaningless for `?`
```

Comment:
wait-- so `$(a),?` is *accepted* but the `,` is just a no-op? seems like it should be an error